spec/services/audits/progress_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+describe Audits::Progress do
```

Comment:
Minor thing, I've been tagging by service system specs with `type: :service` in case we later want to be able to filter them, or run just specific types of tests. Maybe something for the style guide?